newIDE/app/src/SceneEditor/SetupGridDialog.js

```diff
@@ -43,6 +48,18 @@ export default class SetupGridDialog extends Component {
         open={this.props.open}
         onRequestClose={this.props.onCancel}
       >
+        <MiniToolbar>
+          <MiniToolbarText>
+            <Trans>Line color:</Trans>
+          </MiniToolbarText>
+          <ColorPicker
+            color={this.state.gridColor}
+            onChangeComplete={color => {
+              this.setState({ gridColor: color.rgb });
+            }}
+          />
+        </MiniToolbar>
+
         <TextField
           floatingLabelText={<Trans>Cell width (in pixels)</Trans>}
```

Comment:
We should probably group the fields 2 by 2 in a `<ResponsiveLineStackLayout>`, one for the width/height, one for X/Y.